stl/inc/xutility

```diff
@@ -140,7 +140,7 @@ _NODISCARD constexpr _Ty* to_address(_Ty* const _Val) noexcept {
 }
 
 template <class _Ptr>
-_NODISCARD auto to_address(const _Ptr& _Val) noexcept {
+_NODISCARD constexpr auto to_address(const _Ptr& _Val) noexcept {

```

Comment:
Should this get something like:

```
// TRANSITION, LWG issue pending number assignment
```

?

Reply:
Also shouldn't this be `_CONSTEXPR20`?

Reply:
> Also shouldn't this be `_CONSTEXPR20`?

`to_address` is C++20-only.

Reply:
I've used `// Per pending LWG issue` in the past, IIRC.

Reply:
I'll add a bit more detail: `// constexpr per pending LWG issue, submitted 2020-01-14`.